crates/rocksdb-store/src/chain_state/db.rs

```diff
@@ -49,13 +49,18 @@ impl ChainstateDatabase for ChainstateDb {
     }
 
     // TODO: define what toplevel means more clearly
+    // TODO: Why is it called toplevel when it is returning any record based on index?
```

Comment:
Consider in Ethereum where there's a whole state trie of accounts.  We don't load all of those into memory at once.  We can think of that as being the "deep" state.  We don't have deep state like that on the Strata chain right now, but we *might*.  So "toplevel" in this context refers to all of the state that we *can* load into memory unconditionally.

Reply:
Got it, thanks!